### DISCUSSION TITLE
Status bar on mobile?

### DISCUSSION THREAD

**Discussion by hackemateninja** [AvaloniaUI/Avalonia#12661] [Q&A]
I am moving from react native to AvaloniaUI because I see the potential that Avalonia has, but now I am creating mobile apps with F# and AvaloniaUI MVVM, but I see that doesn't exist things like the status bar and optimization and mobile behaviors for textbox when it appears the virtual keyboard, so I would like to know if that would be a possible feature included If it is possible that should solve a lot of inconsistency on mobile designs.

At the moment, I am limiting my design to a black or white background

**Comment by maxkatz6** [comment]
> but I see that doesn't exist things like the status bar and optimization

There is https://docs.avaloniaui.net/docs/next/concepts/services/insets-manager

> mobile behaviors for textbox when it appears the virtual keyboard

As for now, root view size should be reduced to fit the keyboard. A special event to get on-screen keyboard boundaries is planned though.
